src/test/ui/coherence/coherence-fundamental-trait-objects.old.stderr

```diff
@@ -2,9 +2,11 @@ error[E0117]: only traits defined in the current crate can be implemented for ar
   --> $DIR/coherence-fundamental-trait-objects.rs:15:1
    |
 LL | impl Misc for dyn Fundamental<Local> {}
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ impl doesn't use types inside crate
+   | ^^^^^^^^^^^^^^----------------------
+   | |             |
+   | |             `(dyn coherence_fundamental_trait_lib::Fundamental<Local> + 'static)` is not defined in the current crate
```

Comment:
I think this should be ``` `coherence_fundamental_trait_lib::Fundamental<Local>` is not defined in the current crate ```. Having the `dyn` and lifetime bound is confusing (also `+ 'static` is default, so needn't be included).

Reply:
I couldn't remove the `dyn`, but _that_ shouldn't be too big of a problem.